DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -54,6 +95,29 @@ simulated event bool IsMember(PlayerReplicationInfo PRI, optional bool bNoCascad
     return bool(GetMask() & (1 << PRI.VoiceID));
 }
 
+simulated function array<PlayerReplicationInfo> GetMembers()
```

Comment:
For the sake of speed, it might be better to pass `array<PlayerReplicationInfo>` as an `out` variable to be written to by the function rather than returning a brand new list.